artifactory/commands/transferfiles/utils.go

```diff
@@ -682,3 +682,21 @@ func getErrorOrDelayFiles(repoKeys []string, getDirPathFunc func(string) (string
 	}
 	return
 }
+
+// Increments index until the file path is unique.
+func getErrorOrDelayUniqueFilePath(dirPath string, getFileNamePrefix func() string) (delayFilePath string, err error) {
```

Comment:
getErrorOrDelayUniqueFilePath --> getUniqueDelayOrErrorFilePath
(this way, the reader won't think the "error" in the function name refers to an error that the function may return).